packages/create-snap/src/main.test.ts

```diff
@@ -0,0 +1,17 @@
+import * as cliModule from './cli';
+
+jest.mock('./cli', () => ({
+  cli: jest.fn(),
+}));
+
+describe('main', () => {
+  it('executes the CLI application', async () => {
+    await import('./main');
+    expect(cliModule.cli).toHaveBeenCalledTimes(1);
+    expect(cliModule.cli).toHaveBeenCalledWith(process.argv);
+    expect(global.snaps).toStrictEqual({
+      // see test/setup.js
+      verboseErrors: false,
+    });
```

Comment:
I don't think it's necessary to test this, since it's set in the test setup file?